README.md

```diff
@@ -1,7 +1,7 @@
 # swag-for-dev
 
 [![Awesome](https://awesome.re/badge-flat.svg)](https://awesome.re)
-[![Build Status](https://travis-ci.org/swapagarwal/swag-for-dev.svg?branch=master)](https://travis-ci.org/swapagarwal/swag-for-dev)
```

Comment:
i'll go for keeping this line from now as we are not disabling travis right away

Reply:
done